test/lib/apus/adapters/twilio_adapter_test.exs

```diff
@@ -12,14 +12,22 @@ defmodule Apus.TwilioAdapterTest do
         auth_token: "fake-auth-token"
       }
 
-      message = Message.new(from: "+15551234567", to: "+15557654321", body: "Hello there")
+      message =
+        Message.new(
+          from: "+15551234567",
+          to: "+15557654321",
+          body: "Hello there",
+          provider: nil,
+          message_id: nil
+        )
 
       use_cassette "twilio_sms_from_success", match_requests_on: [:request_body] do
         {:ok, %{} = tw_message} = TwilioAdapter.deliver(message, config)
 
         assert tw_message.from == "+15551234567"
         assert tw_message.to == "+15557654321"
         assert tw_message.body == "Hello there"
+        assert !is_nil(tw_message.message_id)
```

Comment:
Can we assert against the `sid` value in the VCR cassette?

If not, we can simplify this slightly using a refute (ie. `refute is_nil(tw_message.message_id)`).